dist/images/ovndb-raft-functions.sh

```diff
@@ -272,12 +335,14 @@ ovsdb-raft() {
   echo "=============== ${db}-ovsdb-raft ========== RUNNING"
 
   if [[ "${POD_NAME}" == "ovnkube-db-0" ]]; then
-    # set the election timer value before other servers join the cluster and it can
-    # only be set on the leader so we must do this in ovnkube-db-0 when it is still
-    # a single-node cluster
-    set_election_timer ${db} ${election_timer}
-    if [[ ${db} == "nb" ]]; then
-      set_northd_probe_interval
+    if ${initial_raft_create}; then
+      # set the election timer value before other servers join the cluster and it can
+      # only be set on the leader so we must do this in ovnkube-db-0 when it is still
+      # a single-node cluster
+      set_election_timer ${db} ${election_timer}
+      if [[ ${db} == "nb" ]]; then
+        set_northd_probe_interval
+      fi
     fi
```

Comment:
Why are we wrapping the calls to change election_timer and northd_probe_interval under initial_raft_create? I think it's ok to just let it be called all the time since if the values have not changed, then the call is no-op.